flink-tests/src/test/java/org/apache/flink/test/misc/MiscellaneousIssuesITCase.java

```diff
@@ -134,7 +106,8 @@ public void testAccumulatorsAfterNoOp() {
 		final String accName = "test_accumulator";
 
 		try {
-			env.setParallelism(6);
+			ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+			env.setParallelism(4);
```

Comment:
i had to reduce the parallelism to 4 as it failed with 6 due to not enough slots.

Reply:
Why not start a mini cluster with 2 TMs and 3 slots as before?

Reply:
yeah, that's probably the better option.